src/python_testing/TC_SU_2_3.py

```
@@ -0,0 +1,446 @@
+        ]
+
+    @async_test_body
+    async def teardown_test(self):
+        if hasattr(self, "current_provider_app_proc") and self.current_provider_app_proc is not None:
+            logger.info("Terminating existing OTA Provider")
+            self.current_provider_app_proc.terminate()
+            self.current_provider_app_proc = None
+        super().teardown_test()
+
+    @async_test_body
+    async def test_TC_SU_2_3(self):
```

Pylint (4.0.6)

[refactor] Too many local variables (28/16)

(R0914)